src/main/java/graphql/schema/TypeVisitor.java

```diff
@@ -0,0 +1,59 @@
+package graphql.schema;
```

Comment:
Naming is hard but I would call this (and the others) GraphqlTypeVisitor

My reasoning is that we have runtime types and AST types and its already confusing

So we have GraphqlType / GraphqlObjectType and so on to help with that

Hence I visitor of GraphqlType thigns should have the GraphqlTypeVisitor naming.  Same with the stubb and so on